share/spice/cryptocurrency/cryptocurrencylist.txt

```diff
@@ -219,7 +219,7 @@ enrg,energycoin,energy coin,
 eqx,equinoxcoin,equinox coin,
 erm,ermiscoin,ermis coin,
 etc,eternalcoin,eternal coin,
-eth,etherium,etherium coin,
+eth,ether,etherium,etherium coin,ethereum,
```

Comment:
Oh, doh! You were right to remove `ethereum`; checking the original issue and some sources it does seem ethereum isn't a currency. We should probably keep `ethereum coin` though.